src/lazy/expanded/mod.rs

```diff
@@ -90,18 +90,18 @@ pub mod template;
 //  happens to be available in the buffer OR the set that leads up to the next encoding directive.
 //  The value proposition of being able to lazily explore multiple top level values concurrently
 //  would need to be proved out first.
-#[derive(Copy, Clone, Debug)]
-pub struct EncodingContext<'top> {
-    pub(crate) macro_table: &'top MacroTable,
-    pub(crate) symbol_table: &'top SymbolTable,
-    pub(crate) allocator: &'top BumpAllocator,
+#[derive(Debug)]
+pub struct EncodingContext {
+    pub(crate) macro_table: MacroTable,
+    pub(crate) symbol_table: SymbolTable,
+    pub(crate) allocator: BumpAllocator,
```

Comment:
🗺️ An earlier PR (#763) replaced the three-field `EncodingContext<'top>` in a variety of types with a single pointer. To do this, it put a copy of the three-field `EncodingContext<'top>` in the bump allocator and then gave out a reference to it.

This PR eliminates a layer of conceptual indirection by having the `EncodingContext` type directly own its three fields. Other types hold an `EncodingContextRef<'top>` that points to the `EncodingContext`.